Userland/Utilities/jail-attach.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    unsigned jail_index = 0;
```

Comment:
Can't we pledge an empty set here?
Totally outside my scope, but we might want to add a `jail` pledge.

Reply:
There's already a `jail` pledge I implemented, so I should use it here, thanks for noticing :)